_posts/2019-03-18-rules-jvm-external-maven.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: posts
+title: "A repository rule for calculating transitive Maven dependencies"
+authors:
+  - jin
+  - dkelmer
+---
+
+We’re delighted to introduce the initial release of
+[`rules_jvm_external`](https://github.com/bazelbuild/rules_jvm_external), a
+repository rule to resolve and fetch artifacts transitively in Maven
+repositories, like Maven Central and Google Maven. This project is officially
+maintained by the Bazel team.
```

Comment:
This is an exciting change. Is there something we could do to make the first sentence stronger? "We are excited to announce that Bazel now has official support for Maven repositories through a new rule set, `rules_jvm_external`. This rule set is an iteration on existing rule sets that makes Maven repositories easier to manage and lets users calculate Maven dependencies during Bazel's execution phase. This means..."

I think it would be good for the first paragraph to have some of this information so that the reader understands right away what to expect from the rest of the post.

Reply:
It is an exciting change!

Reply:
This is less of an iteration on existing rules but more of a fresh approach. I've switched the word "delighted" to "excited", because it's more accurate about how we really feel :smile: 

I think this project is heavily requested enough to the point that I don't think we need a stronger hook, and the current phrasing that goes straight to the point. What do you think?